com/preisposition.go

```diff
@@ -0,0 +1,31 @@
+	Leistungstyp       leistungstyp.Leistungstyp               `json:"leistungstyp" validate:"required"`       // Standardisierte Bezeichnung für die abgerechnete Leistungserbringung. Details siehe ENUM Leistungstyp
+	//Leistungsbezeichung string	// Bezeichnung für die in der Position abgebildete Leistungserbringung
+	Preiseinheit             waehrungseinheit.Waehrungseinheit    `json:"preiseinheit" validate:"required"`                       // Festlegung, mit welcher Preiseinheit abgerechnet wird, z.B. Ct. oder €. Details siehe ENUM Waehrungseinheit
+	Bezugsgroesse            mengeneinheit.Mengeneinheit          `json:"bezugsgroesse" validate:"required"`                      // Hier wird festgelegt, auf welche Bezugsgrösse sich der Preis bezieht, z.B. kWh oder Stück. Details siehe ENUM Mengeneinheit
+	Zeitbasis                *zeiteinheit.Zeiteinheit             `json:"zeitbasis,omitempty" validate:"required"`                // Die Zeit(dauer) auf die sich der Preis bezieht. Z.B. ein Jahr für einen Leistungspreis der in €/kW/Jahr ausgegeben wird. Details siehe ENUM Zeiteinheit
+	Tarifzeit                *tarifzeit.Tarifzeit                 `json:"tarifzeit,omitempty" validate:"required"`                // Festlegung, für welche Tarifzeit der Preis hier festgelegt ist. Siehe ENUM Tarifzeit
+	BDEWArtikelnummer        *bdewartikelnummer.BDEWArtikelnummer `json:"bdewartikelnummer,omitempty" validate:"required"`        // Eine vom BDEW standardisierte Bezeichnug für die abgerechnete Leistungserbringung. Diese Artikelnummer wird auch im Rechnungsteil der INVOIC verwendet. Siehe ENUM BDEWArtikelnummer
+	ArtikelId                *artikelid.ArtikelId                 `json:"artikelid,omitempty" validate:"required"`                //	Standardisierte vom BDEW herausgegebene Liste, welche im Strommarkt die BDEW-Artikelnummer ablöst ENUM ArtikelId
+	Zonungsgroesse           *bemessungsgroesse.Bemessungsgroesse `json:"zonungsgroesse,omitempty" validate:"required"`           // Mit der Menge der hier angegebenen Größe wird die Staffelung/Zonung durchgeführt. Z.B. Vollbenutzungsstunden. Siehe ENUM Bemessungsgroesse
+	FreimengeBlindarbeit     decimal.NullDecimal                  `json:"freimengeBlindarbeit,omitempty" validate:"required"`     // Der Anteil der Menge der Blindarbeit in Prozent von der Wirkarbeit, für die keine Abrechnung erfolgt
+	FreimengeLeistungsfaktor decimal.NullDecimal                  `json:"freimengeLeistungsfaktor,omitempty" validate:"required"` //	Der cos phi (Verhältnis Wirkleistung/Scheinleistung) aus dem die Freimenge für die Blindarbeit berechnet wird als tan phi (Verhältnis Blindleistung/Wirkleistung)
+	Preisstaffeln            []Preisstaffel                       `json:"preisstaffel " validate:"required"`                      //	Preisstaffeln, die zu dieser Preisposition gehören. Details siehe COM Preisstaffel
```

Comment:
```suggestion
	Berechnungsmethode kalkulationsmethode.Kalkulationsmethode `json:"berechnungsmethode" validate:"required"` //	Das Modell, das der Preisbildung zugrunde liegt
	Leistungstyp       leistungstyp.Leistungstyp               `json:"leistungstyp" validate:"required"`       // Standardisierte Bezeichnung für die abgerechnete Leistungserbringung
	//Leistungsbezeichung string	// Bezeichnung für die in der Position abgebildete Leistungserbringung
	Preiseinheit             waehrungseinheit.Waehrungseinheit    `json:"preiseinheit" validate:"required"`                       // Festlegung, mit welcher Preiseinheit abgerechnet wird, z.B. Ct. oder €
	Bezugsgroesse            mengeneinheit.Mengeneinheit          `json:"bezugsgroesse" validate:"required"`                      // Hier wird festgelegt, auf welche Bezugsgrösse sich der Preis bezieht, z.B. kWh oder Stück. Details siehe ENUM Mengeneinheit
	Zeitbasis                *zeiteinheit.Zeiteinheit             `json:"zeitbasis,omitempty" validate:"required"`                // Die Zeit(dauer) auf die sich der Preis bezieht. Z.B. ein Jahr für einen Leistungspreis der in €/kW/Jahr ausgegeben wird
	Tarifzeit                *tarifzeit.Tarifzeit                 `json:"tarifzeit,omitempty" validate:"required"`                // Festlegung, für welche Tarifzeit der Preis hier festgelegt ist
	BDEWArtikelnummer        *bdewartikelnummer.BDEWArtikelnummer `json:"bdewartikelnummer,omitempty" validate:"required"`        // Eine vom BDEW standardisierte Bezeichnug für die abgerechnete Leistungserbringung. Diese Artikelnummer wird auch im Rechnungsteil der INVOIC verwendet
	ArtikelId                *artikelid.ArtikelId                 `json:"artikelid,omitempty" validate:"required"`                //	Standardisierte vom BDEW herausgegebene Liste, welche im Strommarkt die BDEW-Artikelnummer ablöst ENUM ArtikelId
	Zonungsgroesse           *bemessungsgroesse.Bemessungsgroesse `json:"zonungsgroesse,omitempty" validate:"required"`           // Mit der Menge der hier angegebenen Größe wird die Staffelung/Zonung durchgeführt. Z.B. Vollbenutzungsstunden
	FreimengeBlindarbeit     decimal.NullDecimal                  `json:"freimengeBlindarbeit,omitempty" validate:"required"`     // Der Anteil der Menge der Blindarbeit in Prozent von der Wirkarbeit, für die keine Abrechnung erfolgt
	FreimengeLeistungsfaktor decimal.NullDecimal                  `json:"freimengeLeistungsfaktor,omitempty" validate:"required"` //	Der cos phi (Verhältnis Wirkleistung/Scheinleistung) aus dem die Freimenge für die Blindarbeit berechnet wird als tan phi (Verhältnis Blindleistung/Wirkleistung)
	Preisstaffeln            []Preisstaffel                       `json:"preisstaffel " validate:"required"`                      //	Preisstaffeln, die zu dieser Preisposition gehören
```
das ganze details siehe können wir uns in auto-generierten dokus sparen. wird ja in godoc dann sowieso anklickbar sein.